src/SerialDataCollector.cpp

```diff
@@ -746,9 +748,7 @@ namespace splash
     {
         this->fileStatus = FST_CREATING;
 
-        // appends the mpiPosition to the filename (e.g. myfile_0_1_0.h5)
```

Comment:
The getFullFilename functions is properly documented. I added a specific example (like it was here before) there too, but I don't see why we should describe the functions at their usage point.